bindata/network/additional-networks/sriov/sriov-device-plugin.yaml

```diff
@@ -0,0 +1,53 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: sriov-device-plugin
+  namespace: openshift-sriov
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the SR-IOV network device plugin on each node.
+spec:
+  selector:
+    matchLabels:
+      app: sriov-device-plugin
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sriov-device-plugin
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      hostNetwork: true
+      hostPID: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      serviceAccountName: sriov-device-plugin
+      containers:
+      - name: sriov-device-plugin
+        image: {{.SRIOVDevicePluginImage}}
+        args:
+        - --log-level=10
+        securityContext:
+          privileged: true
```

Comment:
why is device plugin privileged? it doesn't seem to require any meddling with the host, it only discovers and advertises devices so I would think it should work without the privilege escalated? (Perhaps it's because of the bug in openshift installer where we don't apply proper SELinux labels to `/var/lib/kubelet/device-plugins` directory? In which case, maybe the right course is fixing the installer / SELinux packages?)

Reply:
Yes, I think the PR upstream that fixing selinux problem of device-plugins dir is not merged yet.

Reply:
You refer to https://github.com/kubernetes/kubernetes/pull/73241 right? Yes it's not merged; if this is the reason why this manifest carries `privileged: true` then maybe we should tag it with a comment as something that we should revisit later when the fix is merged upstream and synced into OpenShift.